.idea/github-bot.iml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
For consistency, can you go into Webstorm, right-click on the project and rename it to `cesium-concierge` so that everything matches.  Thanks.